src/servers/graphql-server.ts

```diff
@@ -148,32 +148,35 @@ export const sessionContext = ({
   return addAttributesToCurrentSpanAndPropagate(
     {
       "token.sub": tokenPayload?.sub,
+      "token.iss": tokenPayload?.iss,
       [SemanticAttributes.HTTP_CLIENT_IP]: ip,
     },
     async () => {
       // note: value should match (ie: "anon") if not an accountId
       // settings from dev/ory/oathkeeper.yml/authenticator/anonymous/config/subjet
-      const maybeUserId = checkedToUserId(tokenPayload?.sub || "")
+      const maybeUserId = checkedToUserId(tokenPayload?.sub ?? "")
+
       if (!(maybeUserId instanceof ValidationError)) {
         const userId = maybeUserId
-
         const account = await Accounts.getAccountFromUserId(userId)
-        if (account instanceof Error) throw mapError(account)
-        domainAccount = account
-
-        // not awaiting on purpose. just updating metadata
-        // TODO: look if this can be a source of memory leaks
-        Accounts.updateAccountIPsInfo({
-          accountId: account.id,
-          ip,
-          logger,
-        })
-
-        const userRes = await UsersRepository().findById(account.kratosUserId)
-        if (userRes instanceof Error) throw mapError(userRes)
-        user = userRes
-
-        addAttributesToCurrentSpan({ [ACCOUNT_USERNAME]: domainAccount?.username })
+        if (account instanceof CouldNotFindAccountFromKratosIdError) {
+          // do nothing, the jwt is valid but the account does not exist
```

Comment:
if we were to move this endpoint https://github.com/GaloyMoney/galoy/pull/2071/files#diff-2a3744191b13e4b2375d4bc44df6706cd453f9b3ccbf3970f801ad4ce97219aaR16 outside of graphql, then we would not need to have this exception. 

what do you think @bodymindarts @ntheile ? could be part of a future refactoring.